torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1407,6 +1407,20 @@ def sample_inputs_baddbmm(op_info, device, dtype, requires_grad, **kwargs):
                                              broadcasts_input=broadcasts_input))
     return tuple(sample_inputs)
 
+def sample_attn(self, device, dtype, requires_grad=False, **kwargs):
+        samples = []
+        shapes = [(2,2), (3,3), (2,3), (10,10), (100,100)]
+        for shape in shapes:
```

Comment:
You only test the case where all 3 inputs have the same shape here?

Reply:
Added more test cases with different shapes below.